roles/neutron-common/handlers/main.yml

```diff
@@ -11,3 +11,7 @@
     - "{{ neutron.services.neutron_linuxbridge_agent }}"
     - "{{ neutron.services.neutron_metadata_agent }}"
     - "{{ neutron.services.neutron_lbaasv2_agent }}"
+
+- name: reload vxlan module
+  shell: rmmod mlx4_en; rmmod vxlan; rmmod mlx4_core; modprobe vxlan ;modprobe mlx4_en;modprobe mlx4_core
```

Comment:
use command instead of shell and convert multiple commands to list and use with_items